Microsoft.Toolkit.Uwp.UI.Controls/ColumnResizer/ColumnResizer.Events.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Windows.UI.Xaml;
+using Windows.UI.Xaml.Input;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// ColumnResizer is a UI control that add the resizing functionality to a Grid Column.
+    /// </summary>
+    public partial class ColumnResizer
+    {
+        /// <summary>
+        /// Parent Grid On Pointer Moved Event
+        /// </summary>
+        private void Resizable_PointerMoved(object sender, PointerRoutedEventArgs e)
```

Comment:
I'm wondering if it would be easier to just use ManipulationDelta instead, since this is fired for drag.